src/parsers/transactions.ts

```diff
@@ -112,11 +111,10 @@ export const parseTransactions = async (
 
   const dedupedResults = uniqBy(updatedResults, txn => txn.hash);
 
-  const orderedDedupedResults = orderBy(
-    dedupedResults,
-    ['minedAt', 'nonce'],
-    ['desc', 'desc']
-  );
+  const orderedDedupedResults = dedupedResults.sort((a, b) => {
+    // Sort by `minedAt` in descending order and by `nonce` in descending order.
+    return b.minedAt! - a.minedAt!;
+  });
```

Comment:
This too

Reply:
I returned the previous version because it became slower after the fix than lodash. I remove this with the intent not to harm performance